src/components/sync-connected-sites.tsx

```diff
@@ -125,14 +125,14 @@ export function SyncConnectedSites( {
 
 								<Button
 									variant="link"
-									className="!text-a8c-gray-70 hover:!text-a8c-blueberry"
+									className="!text-a8c-gray-70 hover:!text-a8c-blueberry truncate"
 									onClick={ () => {
 										getIpcApi().openURL( connectedSite.url );
 									} }
 								>
-									{ connectedSite.url } <ArrowIcon />
+									<span className="truncate">{ connectedSite.url }</span> <ArrowIcon />
```

Comment:
I think this is a good approach, I tried not truncating it and I think it complicates things